source/locale/en/symbols.dic

```diff
@@ -99,8 +99,8 @@ _	line	most
 ‘	left tick	most
 ’	right tick	most
 –	en dash	most	always
-—	em dash	most
-­	soft hyphen	most
+—	em dash	most	norep
+­	soft hyphen	all	norep
```

Comment:
Could you indicate in the PR's description why you have chosen only to change the `preserve` parameter? As described in https://github.com/nvaccess/nvda/issues/13668#issuecomment-1122143209 and in https://github.com/nvaccess/nvda/issues/13668#issuecomment-1171052330, changing the `preserve` parameter allows to fix #13668 only for eSpeak which can handle this character, but not for other synths. To have the issue fixed for other synths, the soft hyphen should be removed from the text passed to the synthesizer.

Reply:
I'm going to reverse the soft hyphen changes, as it is unclear if it should be set to most, all or char.

```suggestion
­	soft hyphen	most
```